awsshell/wizard.py

```diff
@@ -71,6 +71,10 @@ def __init__(self, session=None, interaction_loader=None,
         self._interaction_loader = interaction_loader
         if interaction_loader is None:
             self._interaction_loader = InteractionLoader()
+        if delegation_loader is None:
```

Comment:
Fixed this up. Rather than inject a WizardLoader, I simply register a custom data loader on the session and inject that session instead.